Site/webapp/wdkCustomization/js/client/components/Srt.tsx

```diff
@@ -161,11 +210,18 @@ function SrtForm({
   initialIdsState,
   idsInputHelp,
   projectId,
-  formActionUrl
+  formActionUrl,
+  paramValueStore,
+  selectedSrtForm
 }: SrtFormConfig) {
   const [ idsState, setIdsState ] = useState(initialIdsState);
   const [ formState, updateFormState ] = useState(initialReporterFormState);
 
+  useEffect(() => {
+    paramValueStore?.updateParamValues(`srt/${selectedSrtForm}/ids`, { initialIdsState: idsState })
+    paramValueStore?.updateParamValues(`srt/${selectedSrtForm}`, { initialReporterFormState: JSON.stringify(formState) })
```

Comment:
It might be less redundant to store both the id and form state in a single IndexedDB entry: `srt/${selectedSrtForm}`.

Reply:
Yes, we discussed this during our huddle and I thought there were concerns about needlessly `JSON.parse`'ing and `JSON.stringify`'ing the idsState so I kept them separate. Perhaps I misunderstood that piece of the conversation. It would certainly simplify the construction of the `compatibleSrtConfigs` object.

Reply:
Yup, that's right! In this case, there are no performance penalties for using one IndexedDB entry per form type. I had forgotten that the `idsState` is a simple string (not a JSON object), so you never have to stringify or parse that data.

Reply:
So given that, one IndexedDB entry per form type is the way to go.